project-canvas/src/components/backlog-view/Issue/DetailView.tsx

```diff
@@ -29,21 +36,33 @@ export function DetailView({
   epic,
   labels,
   assignee,
-  creator,
   description,
-  sprintId,
   subtasks,
   created,
   updated,
   comment,
   type,
+  projectId,
+  sprint,
 }: Issue) {
-  const [defaultdescription, setdefaultdescription] = useState(description)
-  const [showInputEle, setShowInputEle] = useState(false)
+  const { data: issueTypes } = useQuery({
+    queryKey: ["issueTypes", projectId],
+    queryFn: () => getIssueTypes(projectId),
+    enabled: !!projectId,
+  })
+  const queryClient = useQueryClient()
+
+  const [defaultStatus, setDefaultStatus] = useState(status)
+  const mutationStatus = useMutation({
```

Comment:
```suggestion
  const mutation = useMutation({
```
this is not the mutation status but the whole mutation object

Reply:
Then that's called statusMutation as it is the mutation of the status not mutationStatus as it is not the current status of the mutation.